### PR TITLE
seding a triggered email

### PR DESCRIPTION
**Pull request by Luitame**
I wrote more one method. The new method [send email](https://developer.salesforce.com/docs/atlas.en-us.mc-apis.meta/mc-apis/messageDefinitionSends.htm) using the Marketing Cloud API.

### CODE REVIEW

**Comment by Luitame** [comment]
[Create an abstraction layer to send emails](https://trello.com/c/1L6al8B4/215-create-an-abstraction-layer-to-send-emails)

File: src/Api/MarketingCloud/Messaging.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+     * In order to use this service, configure a triggered send definition in Email Studio.
+     *
+     * @param string $triggeredSendDefinitionId
+     * @param array  $from
+     * @param array  $to
+     * @param array  $options
+     *
+     * @return array
+     */
+    public function sendEmail(string $triggeredSendDefinitionId, array $from, array $to, array $options): array
```
Comment (ianrodrigues) [inline src/Api/MarketingCloud/Messaging.php]:
```suggestion
    public function sendEmail(string $triggeredSendDefinitionId, array $from, array $to, array $options = []): array
```

Do we really need the `$options` being mandatory?

File: src/Api/MarketingCloud/Messaging.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    {
+        return $this
+            ->marketingCloud
+            ->request(
+                'POST',
+                "/messaging/v1/messageDefinitionSends/{$triggeredSendDefinitionId}/send",
+                [
+                    $from,
+                    $to,
+                    $options,
```
Comment (ianrodrigues) [inline src/Api/MarketingCloud/Messaging.php]: If it's not mandatory, we can use an `array_merge` right here.

```suggestion
                    array_merge(['my_mandatory_options' => 'some_value'], $options),
```

**Review by ianrodrigues — CHANGES_REQUESTED** [review]

File: src/Api/MarketingCloud/Messaging.php (hunk 1 above)
Comment (Luitame) [reply src/Api/MarketingCloud/Messaging.php]: Nope. Good catch. I'll fix it.

File: src/Api/MarketingCloud/Messaging.php (hunk 2 above)
Comment (Luitame) [reply src/Api/MarketingCloud/Messaging.php]: Thanks. Good tip!

**Comment by Luitame** [comment]
@ianrodrigues Can you take a look again?

**Pull request merged**